README.md

```diff
@@ -145,25 +145,33 @@ An object representing an alert session. Contains the following fields:
 
 **alertState (ALERT_STATE):** Thc current alert state of the alert session
 
-**incidentCategory (int):** The integer representing the incident category associated with the alert session
+**incidentCategoryKey (string):** The integer representing the incident category associated with the alert session
```

Comment:
Thanks! Yes, I'll do that.

It's a string. But yeah, right now for both Buttons and ODetect, that string happens to be a stringified integer like `"2"`. So I've updated this comment to just leave out integer entirely because from the library's point of view, this will work with any string.